src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -22,6 +51,48 @@
 
         dot-sub-nav {
             width: $navigation-width-collased;
+
+            ::ng-deep .dot-nav-sub {
+                .dot-nav-sub__link {
+                    color: $black;
+                    padding: $basic-padding $basic-padding * 2;
+                }
+
+                .dot-nav-sub__link {
+                    &.dot-nav-sub__link--active,
+                    &:hover {
+                        background-color: $gray-bg;
+                    }
+                }
+            }
+        }
+
+        &.contextmenu {
+            .dot-nav__item {
+                &:not(.dot-nav__item--active) {
+                    background-color: rgba(var(--color-white_rgb), 0.1);
+                }
+            }
+
+            dot-sub-nav {
+                opacity: 1;
+                position: absolute;
+                left: $navigation-width-collased;
+                width: auto;
+                margin-top: -$navigation-item-height;
+                z-index: 1001;
+                background: $white;
+                box-shadow: $md-shadow-1;
+
+                ::ng-deep .dot-nav-sub {
+                    height: auto !important;
+                    overflow: visible !important;
+                }
+            }
+
+            dot-overlay-mask {
```

Comment:
Shouldn't this be same for the whole component?

Reply:
need's to be removed